### PR TITLE
docs(build): fix typo

### PR DESCRIPTION
**Pull request by tpoisonooo**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

* Fix typo
* Upate FAQ for pip install error
* Add description for ncnn LD_LIBRARY_PATH

Single PR is not worth reviewing, so batching.

## Modification

Update docs and comments.

## BC-breaking (Optional)

no BC-breaking.

## Use cases (Optional)

no need test.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/build/linux.md [hunk 1]
```diff
@@ -84,6 +84,14 @@ conda install pytorch==1.8.0 torchvision==0.9.0 cudatoolkit=11.1 -c pytorch -c c
...
 export torch_version=torch1.8
 pip install mmcv-full==1.4.0 -f https://download.openmmlab.com/mmcv/dist/${cu_version}/${torch_version}/index.html
+</code></pre>
+    </td>
+  </tr>
+  <tr>
+    <td>onnx </td>
+    <td>Install onnx as follows. Refer to the <a href="https://github.com/onnx/onnx">guide</a> for details.
+<pre><code>
+conda install -c conda-forge onnx
```
Comment (AllentDan) [inline docs/en/build/linux.md:94]: well, as conda always installs packages not wanted, `pip` is preferred I guess

Comment (tpoisonooo) [reply docs/en/build/linux.md:94]: This line comes from https://github.com/onnx/onnx README.md ...

Comment (AllentDan) [reply docs/en/build/linux.md:94]: ONNX is actually in [requirements](https://github.com/open-mmlab/mmdeploy/blob/master/requirements/runtime.txt)

File: docs/en/build/linux.md [hunk 2]
```diff
@@ -445,9 +453,11 @@ You can also activate other engines after the model.
 
 #### Build Demo
 
+Take ncnn backend as example:
+
 ```Bash
 cd ${MMDEPLOY_DIR}/build/install/example
 mkdir -p build && cd build
-cmake .. -DMMDeploy_DIR=${MMDEPLOY_DIR}/build/install/lib/cmake/MMDeploy
+cmake .. -DMMDeploy_DIR=${MMDEPLOY_DIR}/build/install/lib/cmake/MMDeploy -Dncnn_DIR=${NCNN_DIR}/build/install/lib/cmake/ncnn
```
Comment (grimoire) [inline docs/en/build/linux.md:461]: Is it necessary to add ncnn library when we use the built SDK? ncnn should have been linked into MMDeploy SDK when we build it. Add ncnn lib to `LD_LIBRARY_PATH` should be enough(or if it is not, we should make it.)

File: docs/en/build/linux.md (hunk 1 above)
Comment (tpoisonooo) [reply docs/en/build/linux.md]: reverted.

File: docs/en/build/linux.md (hunk 2 above)
Comment (tpoisonooo) [reply docs/en/build/linux.md]: done.

**Review by grimoire — APPROVED** [review]
LGTM

**Review by AllentDan — APPROVED** [review]
LGTM

**Pull request merged**